lib/stitches/generator_files/db/migrate/add_enabled_to_api_clients.rb

```diff
@@ -0,0 +1,7 @@
+class AddEnabledToApiClients < ActiveRecord::Migration
+  def change
```

Comment:
@jlambert121 this is so that existing stitches users can run this generator and get their api_clients table updated to work with this PR.  i.e. it makes the transition from old to new eaiser.